File: CS50x/week1/mario(more)/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    while (true)
    {
        //Lets take the input
        int input = get_int("Height: ");
        if (input > 0 && input < 9)
        {
            for (int i = 1; i <= input; i++)
            {
                for (int j = 1; j <= input - i; j++)
                {
                    printf(" ");
                }
            
                for (int j = 1; j <= i ; j++)
                {
                    printf("#");
                }
        
                printf("  ");
            
                for (int j = 1; j <= i ; j++)
                {
                    printf("#");
                }
                
                printf("\n");
    
            }
            return 0;
        }
    }
}

File: CS50x/week6/mario(more)/mario.py
while True:
    n = input("Height: ")
    try:
        n = int(n)
        if n > 0 and n < 9:
            break
    except:
        pass

for i in range(n):
    x = " " * (n - i - 1)
    y = "#" * (i + 1) + "  " + "#" * (i + 1)
    print(x + y)